### PR TITLE
name refactoring , example created for dvir for how to handle state c…

### PR DESCRIPTION
**Pull request by AndreyMich**
Hi Dvir,
I`ve created example for state change please confirm that this is a way that we would continue

### CODE REVIEW

**Comment by DvirBar** [comment]
Hi, that was originally the idea. However I think we're getting it too complicated. After further reading the docs, I came across these:
https://formik.org/docs/guides/arrays
https://formik.org/docs/api/fieldarray

These should make data manipulation easier through formik internals without messing with inner state of formik which might be unpredictable.

File: src/cmps/multi-step-form.jsx [hunk 1]
```diff
@@ -3,40 +3,52 @@ import { addUser } from "../store/user.actions";
...
+
+	return (
+		<div className='form'>
+			<Formik
+				// validationSchema={stepOneValidationSchema}
+				initialValues={data}
+				onSubmit={handleSubmit}
+				validationSchema={validationSchema}
+			>
+				{(formikProps) => {
```
Comment (DvirBar) [inline src/cmps/multi-step-form.jsx]: You can also destructure it like so:
`{({values}) => {.....`

It'll prevent a lot of nesting (:

Comment (AndreyMich) [reply src/cmps/multi-step-form.jsx]: Thanks for review,
The purpose was to pass this props to child components, some of them are using the values and some of them are using other properties/methods that Formik exposes through it child params.
I think we need to schedule a meeting for the next steps in order to decide approach for the architecture that we would be using.

**Comment by AndreyMich** [comment]
@DvirBar I`ve added hebrew dates components + added corresponding attributes in data object

**Pull request merged**